mediawiki/LocalSettings.php

```diff
@@ -36,26 +36,17 @@
 
 ## The URL paths to the logo.  Make sure you change this from the default,
 ## or else you'll overwrite your logo when you upgrade!
-#$wgLogos = [
-#	'1x' => "/themes/images/logo_wikifavelas2024.svg",
-#	'wordmark' => [
-#		"src" => "/themes/images/logo_wikifavelas2024.svg",
-#		"width" => 119,
-#		"height" => 18,
-#	],
-#	'icon' => "/themes/images/logo_wikifavelas.svg",
-#];
-$wgLogos = [ '1x' => "$wgResourceBasePath/themes/images/logo_wikifavelas2024.svg" ];
-
-
+$wgLogos = [
+	'1x' => "$wgResourceBasePath/themes/images/logo_white.png",
+	'icon' => "$wgResourceBasePath/resources/assets/change-your-logo-icon.svg",
+];
 
 ## UPO means: this is also a user preference option
 
 $wgEnableEmail = true;
 $wgEnableUserEmail = true; # UPO
 
-$wgEmergencyContact = "wikifavelas@fiocruz.br";
-$wgPasswordSender = "wikifavelas@fiocruz.br";
+$wgEmergencyContact = "";
```

Comment:
Aqui mesma coisa das environments, isto poderia vir da env, se a env estiver vazia, vai dar o mesmo resultado que colocar uma string vazia:


```suggestion
$wgEmergencyContact = getenv('WG_EMERGENCY_CONTACT');
$wgPasswordSender = getenv('WG_PASSWORD_SENDER');
```